lib/ic/microcontroller/stm32f0xxgxux.yaml

```diff
@@ -0,0 +1,35 @@
+datasheet: http://www.st.com/resource/en/datasheet/stm32f042g6.pdf 
+description: STM32F0 28 pin UQFPN package
+footprint: agg:QFN-28-ST
+name: STM32F0xxGxUx
+ordercodes:
+- [Farnell, '2469551']
+pins:
+- - - [VDD, 17, pwrin]
+    - [VDDIO2, 18, pwrin]
+    - [VDDA, 5, pwrin]
+    - [VSS, 16, pwrin]
+  - - [NRST, 4, in]
+    - [BOOT0, 1, in]
+  - - [PF0, 2, bidi]
+    - [PF1, 3, bidi]
```

Comment:
Please include that these are OSC_IN and OSC_OUT: `PF0-OSC_IN` and `PF1-OSC_OUT`.